src/utils/apiEndpoint.js

```diff
@@ -10,7 +10,7 @@ const getApiEndpoint = (location) => {
       return 'http://localhost:3000';
     }
 
-    return 'https://api.scp.biomage.net';
+    return process.env.API_ENDPOINT_PRODUCTION || 'https://api.scp.biomage.net';
```

Comment:
where is this `API_ENDPOINT_PRODUCTION` defined? I can see that the ui pod has the `DOMAIN_NAME`  environment variable. Use DOMAIN_NAME instead, adding in `api` at the front of it.